apps/keyboard/js/views/swipeable_panel_view.js

```diff
@@ -28,12 +30,22 @@ SwipeablePanelView.prototype.KEY_COUNT_PER_PANEL = 18;
 
 SwipeablePanelView.prototype.SWIPE_THRESHOLD = 20;
 
+SwipeablePanelView.prototype.SWIPE_FRACTION = 0.25;
+
+SwipeablePanelView.prototype.SWIPE_SPEED = 0.5;
+
 SwipeablePanelView.prototype.render = function() {
   var panel = document.createElement('div');
   panel.classList.add('swipe-panel');
 
   panel.addEventListener('touchstart', this);
 
+  this.swipingDetector = new SwipingDetector(panel);
+  this.swipingDetector.start();
+
+  panel.addEventListener('pan', this);
```

Comment:
We are not using DOMEvents in Keyboard app. Please use `this.swipingDetector.onpan`, and `this.swipingDetector.onswipe` instead.